source/computer/linux101.rst

```diff
@@ -49,15 +49,236 @@ Linux 入门
 熟悉 Linux 文件系统
 ^^^^^^^^^^^^^^^^^^^
 
+Linux 文件系统就像一颗树一样，从 :file:`/` 目录开始，这个特殊的目录称为根目录。
+一般情况下，根目录下一般有 :file:`/home`\ 、\ :file:`/root`\ 、\ :file:`/bin`\ 、
+:file:`/usr`\ 、\ :file:`/lib`\ 、\ :file:`/opt` 等目录。根目录下的这些目录的具体含义
+由文件系统层次结构标准决定，大部分 Linux 发行版遵循此标准，或由此标准做了细小的调整。例如，
+
+-  :file:`/home` 目录：用户的家目录，存储用户自己的信息
+-  :file:`/bin` 目录：存储必须的程序和命令，所有用户都可用
+-  :file:`/usr` 目录：大多数软件的安装位置
+-  :file:`/opt` 目录：某些闭源或商业软件（如 Matlab、Google Chrome、Google Earth）
+   默认将软件安装到此目录下
+
+日常科研中，我们大多数情况下只在 :file:`/home` 目录下工作。假设用户名是 seismo-learn，
+该用户的家目录便是 :file:`/home/seismo-learn`。在 Linux 系统下，可用 :file:`~` 代表
+家目录。Linux 系统安装后，自动创建的家目录下的目录有 :file:`~/Desktop`\ 、\ :file:`~/Downloads`\ 、
+:file:`~/Documents` 等。熟悉 Linux 系统后可以参考\
+:doc:`《文件管理实践经验》<best-practices/file-organization>`\ 和\
+:doc:`《软件安装实践经验》<best-practices/software-installation>`\
+进一步组织与管理家目录，以提高工作效率。
+
 操作文件与目录
 ^^^^^^^^^^^^^^
 
+打开终端，使用以下命令熟悉和掌握 Linux 系统下文件和目录的常用操作。假设用户名是 seismo-learn。
+
+查看文件和目录::
+
+    # 使用 pwd（print working directory，即打印当前工作目录）命令查看当前所在目录
+    # 可以看出启动终端后，默认进入家目录
+    $ pwd
+    /home/seismo-learn
+
+    # 使用 ls（list，即列表）命令显示当前所在目录（即家目录）含有的子目录和文件
+    $ ls
+    Desktop    Documents    Downloads
+    # 查看 ~/Desktop 目录和 /etc/passwd 文件
+    $ ls Desktop
+    $ ls /etc/passwd
+
+切换目录::
+
+    # 使用 cd（change directory，即切换目录）命令进入根目录
+    $ cd /
+    # 查看根目录中的子目录和文件
+    $ ls
+    bin   etc   lib    mnt  proc  run   srv  tmp  var
+    boot  dev   home  lib64  media   opt  root  sbin  sys  usr
+    # 切换回家目录（以下任一操作均可）
+    $ cd /home/seismo-learn
+    $ cd ~
+    $ cd
+
+新建文件和目录::
+
+    # 进入家目录
+    $ cd ~
+    # 使用 mkdir（make directory，即创建目录）命令新建以下目录
+    $ mkdir codes software workspace
+    # 在 workspace 目录中新建 source 目录
+    $ mkdir workspace/source
+
+    # 进入 workspace 目录
+    $ cd workspace
+    # 使用 touch 命令创建新文件 hello-world.md 和 seiso-learn.md
+    $ touch hello-world.md seiso-learn.md
+
+以下所有操作都假设读者都已先切换到 :file:`~/workspace` 目录下了，即::
+
+    # 进入 ~/workspace 目录
+    $ cd ~/workspace
+
+复制文件和目录::
+
+    # 使用 cp（copy，即复制）命令复制 hello-world.md 文件到同一目录下，并重命名为 hello-world-cp.md
+    $ cp hello-world.md hello-world-cp.md
+    # 复制 hello-world.md 和 seiso-learn.md 文件到 source 目录下
+    $ cp hello-world.md seiso-learn.md source
```

Comment:
Add `ls` commands to check the results of each operation.

Same to all other commands.

Reply:
done in ee9cc03.